src/SfxWeb/src/app/views/cluster/essentials/essentials.component.ts

```diff
@@ -91,6 +98,17 @@ export class EssentialsComponent extends BaseControllerDirective {
       this.nodes.refresh(messageHandler).pipe(map(() => {this.updateItemInEssentials(); })),
       this.systemApp.refresh(messageHandler).pipe(catchError(err => of(null))),
       this.clusterUpgradeProgress.refresh(messageHandler),
+      this.RestClient.getPartitionById("00000000-0000-0000-0000-000000000001", messageHandler).pipe(map((partition) => {
```

Comment:
We can probably turn this getPartitionById into its own function in data.service.ts as getFailoverManagerPartition, or alternatively add a comment explaining that the parititon id 00000000-0000-0000-0000-000000000001 points to FM.